resilience4j-micrometer/src/main/java/io/github/resilience4j/micrometer/tagged/AbstractCircuitBreakerMetrics.java

```diff
@@ -59,6 +60,11 @@ protected void addMetrics(MeterRegistry meterRegistry, CircuitBreaker circuitBre
                 .tag(TagNames.NAME, circuitBreaker.getName())
                 .tag(TagNames.KIND, KIND_SUCCESSFUL)
                 .register(meterRegistry).getId());
+        idSet.add(Gauge.builder(names.getBufferedCallsMetricName(), circuitBreaker, cb -> cb.getMetrics().getNumberOfSlowCalls())
```

Comment:
Unfortunately this doesn't work.
Our sliding window stores successful and failed calls. A successful and a failed call can be fast or slow.
Means: totalNumberOfCalls = numberOfFailedCalls + numberOfSuccessfulCalls

When you visualize the metric `resilience4j.circuitbreaker.buffered.calls` without a tag filter in Grafana, it should show the total number of calls. If we add  the number of slow calls to it, we falsify the total number of calls.

The current number of slow calls must either be a separate metric like `resilience4j.circuitbreaker.slow.calls` or we have to change our sliding window and store `successful`, `successful_slow`, `failed` and `failed_slow`.

Reply:
Oh I understand. I think store `successful`, `successful_slow`, `failed` and `failed_slow` is better. What do you think?

Reply:
Now it seems to be smarter. I wanted to reduce the amount of integer additions/substractions to the minimum in order to improve performance in [AbstractAggregation](https://github.com/resilience4j/resilience4j/blob/master/resilience4j-core/src/main/java/io/github/resilience4j/core/metrics/AbstractAggregation.java) and  [TotalAggregation](https://github.com/resilience4j/resilience4j/blob/master/resilience4j-core/src/main/java/io/github/resilience4j/core/metrics/TotalAggregation.java).

Actually even the number of successful calls isn't stored. Only `totalNumberOfCalls`, `numberOfFailedCalls` and `numberOfSlowCalls`. If we store `successful`, `successful_slow`, `failed` and `failed_slow`, you even always have to add `successful_slow + failed_slow` on every call to calculate the slow_call_rate. Another integer addition which I tried to avoid.
Questions is now. What is more important to us. Metrics or Performance?

Reply:
I think Performance is more important. 
but.. what do you think count like below? 

```java
class AbstractAggregation {
    long totalDurationInMillis = 0;
    int numberOfSlowCalls = 0;
    int numberOfFailedCalls = 0;
    int numberOfCalls = 0;

    // Two integer added.
    int numberOfSuccessSlowCalls = 0;
    int numberOfFailedSlowCalls = 0;

}
```

We don't have to calculate `successful_slow + failed_slow ` on every calls. It is reasonable for performance I think.

Reply:
I think it would be sufficient to add `numberOfFailedSlowCalls`.
The performance to calculate the `slowCallRate` and `failureRate` in [SnapshotImpl](https://github.com/resilience4j/resilience4j/blob/master/resilience4j-core/src/main/java/io/github/resilience4j/core/metrics/SnapshotImpl.java) would be unchanged.

And we can calculate `numberOfSuccessSlowCalls = numberOfSlowCalls - numberOfFailedSlowCalls`, only when needed.

Reply:
Oh.. You are right. It would be better. I'll change that.

Reply:
But still, if we change that. How should the metrics look like?
If we change the names of the tag, it's not backward-compatible anymore.
What is your suggestion?

Reply:
Should we introduce a new metric like `resilience4j.circuitbreaker.slow.calls` with tag `KIND` and values:  `successful` and `failed`?

Reply:
Yes. I think it is good. But `successful` can cause misunderstanding. It can be thought as NOT slow calls. Full naming is better like `SLOW_SUCCESS` and `SLOW_FAILED`.